### PR TITLE
other ns

### PR DESCRIPTION
**Pull request by pwright**
Addresses #159

### CODE REVIEW

File: modules/cli/pages/index.adoc [hunk 1]
```diff
@@ -348,15 +348,29 @@ $ skupper service bind hello-world-backend deployment hello-world-backend
 
 This section shows how to expose a service from a namespace where {skupper-name} is not deployed.
 
+{skupper-name} allows you expose services from other namespaces by default.
```
Comment (grs) [inline modules/cli/pages/index.adoc:351]: This sentence seems wrong to me. The following sentence makes more sense.

File: modules/cli/pages/index.adoc [hunk 2]
```diff
@@ -348,15 +348,29 @@ $ skupper service bind hello-world-backend deployment hello-world-backend
...
+
 .Prerequisites
 
 * A namespace where {skupper-name} is deployed.
-* A service running in a different namespace.
 * A network policy that allows communication between the namespaces
+* cluster-admin permissions if you want to expose resources other than services
+
+
+. Create a site with cluster permissions if you want to expose a Kubernetes resource other than a service:
```
Comment (grs) [inline modules/cli/pages/index.adoc:361]: This should be `if you want to expose a workload in a namespace other than that which this skupper site is associated with` or similar.

File: modules/cli/pages/index.adoc [hunk 3]
```diff
@@ -348,15 +348,29 @@ $ skupper service bind hello-world-backend deployment hello-world-backend
...
+$ skupper init --enable-cluster-permissions
+----
+--
 
-. Log into your cluster and change to the namespace for your site.
 
 . Expose the service on the {service-network}:
+
+
+.. If you want to expose a service, for example, if you did not create a site with `--enable-cluster-permissions`:
```
Comment (grs) [inline modules/cli/pages/index.adoc:373]: You can expose a service with a namespace qualification without requiring the extra cluster permissions. You only need those extra permissions if you want to expose a specific workload type (deployment, statefulset ect) in another namespace. The reason being that skupper then needs to be able to track the pods for that workload individually, hence the need for extra permissions.

File: modules/cli/pages/index.adoc (hunk 2 above)
Comment (pwright) [reply modules/cli/pages/index.adoc]: Do we still want users to 'skupper expose service <service>.<namespace>' without `--enable-cluster-permissions`? 
That's what I'm try to say here, skip this step if you only need to expose a service. 
I've tried to address this by introducing workloads in introduction and then use it here

File: modules/cli/pages/index.adoc (hunk 1 above)
Comment (pwright) [reply modules/cli/pages/index.adoc]: Tried to address this and other comment, maybe easier to review this preview?
image

File: modules/cli/pages/index.adoc (hunk 3 above)
Comment (pwright) [reply modules/cli/pages/index.adoc]: trying to capture this, but also avoiding explaining how it works

**Review by grs — APPROVED** [review]

**Pull request merged**